blockchain/types/block.go

```diff
@@ -328,6 +341,24 @@ func (b *Block) TxHash() common.Hash        { return b.header.TxHash }
 func (b *Block) ReceiptHash() common.Hash   { return b.header.ReceiptHash }
 func (b *Block) Extra() []byte              { return common.CopyBytes(b.header.Extra) }
 
+func (b *Block) ExcessBlobGas() *uint64 {
+	var excessBlobGas *uint64
+	if b.header.ExcessBlobGas != nil {
+		excessBlobGas = new(uint64)
+		*excessBlobGas = *b.header.ExcessBlobGas
+	}
+	return excessBlobGas
+}
+
+func (b *Block) BlobGasUsed() *uint64 {
+	var blobGasUsed *uint64
+	if b.header.BlobGasUsed != nil {
+		blobGasUsed = new(uint64)
+		*blobGasUsed = *b.header.BlobGasUsed
+	}
+	return blobGasUsed
+}
```

Comment:
How's about next code?
```suggestion
func (b *Block) ExcessBlobGas() *uint64 {
	if b.header.ExcessBlobGas == nil {
		return nil
	}
	excessBlobGas := *b.header.ExcessBlobGas
	return &excessBlobGas
}

func (b *Block) BlobGasUsed() *uint64 {
	if b.header.BlobGasUsed == nil {
		return nil
	}
	blobGasUsed := *b.header.BlobGasUsed
	return &blobGasUsed
}
```